rskj-core/src/test/java/co/rsk/net/BlockNodeInformationTest.java

```diff
@@ -37,38 +37,6 @@ private NodeID createNodeID(int i) {
         return new NodeID(ByteBuffer.allocate(4).putInt(i).array());
     }
 
-    @Test
-    public void nodeEvictionPolicy() {
```

Comment:
I removed this test because it checked the behaviour of the getBlocksByNode method and the underlying map, which is now gone